_pages/about-us.md

```diff
@@ -7,5 +7,15 @@ header:
  overlay_filter: "1"
 excerpt: "Learn more about OpenPrinting"
 ---
+## A Brief History
 
-Page under construction
\ No newline at end of file
+OpenPrinting Workgroup was initially established under Free Standards Group(FSG). In July 2006 Linuxprinting.org merged with FSG's OP WorkGroup. On January 22, 2007, the FSG and the Open Source Development Labs(OSDL) merged to form **The Linux Foundation**. Now, OpenPrinting is an opensource organization under The Linux Foundation.
+
+## Major Roles
+
+OpenPrinting:
+* works on the development of new printing architectures, technologies, printing infrastructure, and interface standards for Linux and UNIX-style operating systems. 
+* collaborates with IEEE-ISTO Printer Working Group (PWG) on IPP projects. 
+* is working with SANE to make IPP scanning a reality.
+* maintains cups-filters which allows CUPS to be used on any Unix-based (non-macOS) system.
+* is responsible for the Foomatic database and Common Print Dialog Backends project.
```

Comment:
Make this two lines, one for Foomatic, one for Common Print Dialog Backends.

Reply:
The issue has been corrected.